src/configure/utilities/templateConverter.ts

```diff
@@ -0,0 +1,106 @@
+const templateParametersRegex = /(inputs|variables|assets|secrets)\.\w+/g;
```

Comment:
we also have `system`. Is that not being used in configuration mustache expressions?

Reply:
I haven't seen it yet. I have added it in parameters regex.